lib/web_ui/lib/src/engine/bitmap_canvas.dart

```diff
@@ -478,8 +469,8 @@ class BitmapCanvas extends EngineCanvas {
       if (requiresClipping) {
         restore();
       }
+      _closeCurrentCanvas();
```

Comment:
instead you could remove _closeCurrentCanvas on line 446 in if clause.

Reply:
Hmm, I'm actually not sure what this change is about. Must have been a bad merge 🤔 

Let me check.

Reply:
Ah, right, this was a drive-by change to avoid double-closing. Yes, I will remove the other one. Good catch.

Reply:
Done.